### DISCUSSION TITLE
ImportError: cannot import name 'isin' from

### DISCUSSION THREAD

**Discussion by dsm-72** [google/flax#2365] [Q&A] [answered]
Trying to use jax. Jax imports flax. Flax imports isin from the linen/module.py as `from jax._src.numpy.lax_numpy import isin`. According to StackOverflow that is the imporper way to do this. In any case it throws an error

**Comment by cgarciae** [comment, marked as answer]
Hey @dsm-72! This is just a mismatch between the jax version and the flax version. Try upgrading both JAX and Flax:

```
pip install -U jax flax
```

Hope it helps!
